app/assets/javascripts/shared/datatable/main.js

```diff
@@ -135,10 +135,12 @@ class DradisDatatable {
     this.unbindDataTable();
   }
 
-  toggleLoadingState(rows, isLoading, buttonName) {
-    var button = this.dataTable.buttons(buttonName + ':name');
+  toggleLoadingState(rows, isLoading) {
+    var buttons = this.dataTable.buttons('button');
 
-    $(button[0].node).toggleClass('disabled', isLoading);
+    for (var i = 1; i < buttons.length - 1; i++) {
+      $(buttons[i].node).toggleClass('disabled', isLoading);
+    }
```

Comment:
Can this be used here instead: https://datatables.net/reference/api/buttons().nodes() ? Something like: `buttons.nodes().toggleClass('disabled', isLoading);`